python/tvm/rpc/server.py

```diff
@@ -319,6 +319,8 @@ def __init__(
         load_library=None,
         custom_addr=None,
         silent=False,
+        reuse_addr=False,
```

Comment:
i think we pretty much always want to `reuse_addr`, you could just make this the default